workscheduler/applications/web/controllers/teams.py

```diff
@@ -37,19 +37,20 @@ def show_append_page():
 @login_required
 @admin_required
 def store_team_category():
-    response = Response()
-
     session = get_db_session()
     try:
-        TeamCommandAdapter(session).append_team_category(TeamCategoryForm())
+        team_category_id = UuidFactory.new_uuid()
```

Comment:
why would you use uuid factory here? can you do it on object factory?

Reply:
I'm sorry, I realized that it isn't necessary to use uuid factory here to pass the stored team category id to the edit page.
So I will do it on object factory.